src/email_generator_logic/test_email_generator.py

```diff
@@ -0,0 +1,28 @@
+import email_generator
```

Comment:
jeśli jest to plik testowy to powinien być faktycznym testem. zamiast printów powinny być `assert'y`. Fajnie że jest, ale narazie nie da się tym automatycznie niczego przetestować. Docelowo powinien trafić do src/generator/tests.
Pamiętajmy że przypadkiem użycia tego gagatka jest:

1. zarejestrowanie się nim(czyli stworzenie go)
2. odebranie od niego wiadomości potwierdzającej(czyli na potrzeby testów samo odczytanie zawartości)

Na potrzeby testów można z jednego 1 sec maila wysyłać maile do innych i tak to testować. Dużą niewiadomą jest również sposób potwierdzania rejestracji w takim fakeyou czy elevenlabs, bo boję się że tam będzie trzeba klikać jakiś link "potwierdź logowanie".

Reply:
ogólnie to:
- ElevenLabs potrzebuje potwierdzenia ale mogę wydobyć href i walnąć request na dany adres + BONUS:
https://api.elevenlabs.io/docs#/
"This is the documentation for the ElevenLabs API. You can use this API to use our service programmatically, this is done by using your xi-api-key.
You can view your xi-api-key using the 'Profile' tab on https://elevenlabs.io. Our API is experimental so all endpoints are subject to change."
- CorentinJ jest chyba lokalny nie wiem co z nim robimy ale robi mel spektrogramy  XD (Ja bym się z tego wycofał tbh)
- fakeYou daje radę z loginem bez problemy bo nie wysyłają potwierdzenia 💯 + BONUS:
https://github.com/shards-7/fakeyou.py
https://docs.fakeyou.com/#/